content/en/mobile_testing/mobile_app_tests/advanced_options.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Advanced Options for Mobile App Test Steps
+kind: documentation
+description: Configure advanced options for Mobile Test Steps
+further_reading:
+- link: "https://www.datadoghq.com/blog/test-maintenance-best-practices/"
+  tag: "Blog"
+  text: "Best practices for maintaining end-to-end tests"
+- link: "/mobile_testing/mobile_app_tests/"
+  tag: "Documentation"
+  text: "Learn how to create mobile app tests"
+- link: "/mobile_testing/mobile_app_tests/steps/"
+  tag: "Documentation"
+  text: "Learn how to create steps in mobile app tests"
+- link: "/data_security/synthetics/"
+  tag: "Documentation"
+  text: "Learn about Synthetic Monitoring Data Security"
+---
+
+## Overview
+
+This page describes advanced options for Synthetic mobile app tests. 
+
+## Locate an element
```

Comment:
I think this should be it's own page describing how our multi-locator algorithm works instead of under Advanced Options. 

Users have no way of controlling this behavior, i.e. it's not actually an advanced "option", but it's still helpful for them to know that we have this capability and generally how it works.